File: validator/validator/src/main/java/data/Fans.java
package data;

public class Fans {
    private int goblins, trolls, elves, nifflers, wombats;

    public String fansValid() {

        String sOutputString = "";//erzeugt leeren Diagnostikstring

        //erster check: ist jede Rasse ein- oder mehrmals vertreten?
        if (goblins == 0 || trolls == 0 || elves == 0 || nifflers == 0 || wombats == 0) {
            sOutputString = sOutputString + " You need to have at least one Fan of each Race!" + "\n";
        }

        //zweiter check: max anzahl fans eingehalten?
        if (goblins + trolls + elves + nifflers + wombats != 7) {
            sOutputString = sOutputString + " You need to have exactly 7 Fans!" + "\n";
        }
        return sOutputString;
    }
}

File: Client/Client/Assets/Scripts/MenuScripts/TeamConfig/Fans.cs
using System.Collections;
using System.Collections.Generic;

[System.Serializable]
public class Fans
{
    public int goblins { get; set; }
    public int trolls { get; set; }
    public int elfs { get; set; }
    public int niffler { get; set; }
    public int wombats { get; set; }

    public Fans(int g,int t, int e, int n, int w)
    {
        goblins = g;
        trolls = t;
        elfs = e;
        niffler = n;
        wombats = w;
    }
}
